README.md

```diff
@@ -112,3 +112,62 @@ Add a listener for request
 ```
 client.eventSource.on('requests', function (err, res))
 ```
+
+
+### Managing Child/Related Devices
+
+Related Device accessor
+
+```
+client.relatedDeviceMap[{deviceId}]
+```
+
+Add a Child device
+
+```
+client.relatedDevices.add({deviceId}, {deviceType}, {physicalId})
+```
+
+Remove a Child device
+
+```
+client.relatedDevices.remove({deviceId}, {deviceType}, {physicalId})
+```
+
+Update Related Device State (publish)
+
+```
+client.relatedDeivcesMap[{deviceId}].request.publish({method}, {params})
+```
+
+Update Related Device State (RPC)
+
+```
+client.relatedDeivcesMap[{deviceId}].request.rpc({method}, {params})
+```
+
+Related Device Management
+
+Get Related devices from cloud
+
+```
+client.relatedDeviceManager.getRelatedDevices()
+```
+
+Get Related device Ids from cloud
+
+```
+client.relatedDeviceManager.getRelatedDeviceIds()
+```
+
+Get Related device
+
+```
+client.relatedDeviceManager.getRelatedDevice({Id})
```

Comment:
`getRelatedDevices` vs `getRelatedDevice` ?

Reply:
it gets a singe device by id

Reply:
👍